src/Test/L0/Plugin/TestResultParser/PipelineTestRunPublisherL0.cs

```diff
@@ -90,7 +90,7 @@ public async Task PipelineTestRunPublisher_PublishTestRun_ForBatchedResults()
                     x.UpdateTestRunAsync(It.IsAny<RunUpdateModel>(), It.IsAny<Guid>(), It.IsAny<int>(), null, It.IsAny<CancellationToken>()))
                 .Returns(Task.FromResult(new Microsoft.TeamFoundation.TestManagement.WebApi.TestRun()));
 
-            var publisher = new PipelineTestRunPublisher(clientFactory.Object, pipelineConfig, logger.Object, telemetry.Object) {BatchSize = 3};
+            var publisher = new PipelineTestRunPublisher(clientFactory.Object, pipelineConfig, logger.Object, telemetry.Object) { BatchSize = 3 };
             await publisher.PublishAsync(new TestRun("FakeTestResultParser/1", "Fake", 1)
```

Comment:
can we add a test where no test results are present?

This is for the issue we caught today